src/bn_accounts.erl

```diff
@@ -15,21 +15,24 @@
 
 handle_rpc(<<"account_get">>, {Param}) ->
     Chain = blockchain_worker:blockchain(),
-    case ?jsonrpc_get_param(<<"height">>, Param, false) of
-        false ->
-            Ledger = blockchain:ledger(Chain),
-            Height = bn_txns:follower_height();
+    Height = case ?jsonrpc_get_param(<<"height">>, Param, false) of
```

Comment:
Next step.. instead of Height = use:

```
{Height, Ledger} = case ?jsonrpc_get_param(<<"height">>, Param, false) of
    false -> {bn_txns:follower_height(), blockchain:ledger(Chain)};
   V -> .. % get ledger_at make tuple with V
end,
```

Reply:
apologies for the atrocious formatting.. I use erlfmr pretty extensively in vscode